### ISSUE TITLE
Processing Non spatial elements

### ISSUE THREAD

**Issue by jlaiph** [Autodesk/revit-ifc#440]
I have a less than 1GB  structural elements .rvt file. Whenever exporting it in IFC2X3 coordinate view in Revit 2021, it runs into "Processing Non spatial elements" and freeze there. Is this relate to an unresolvable geometry issue or simply I need to simplify the geometry?

**Comment by jlaiph** [comment]
I was told that Revit somehow prevent such IFC export while possibly the other designers are still working on the model. Any workaround about this?

**Comment by jlaiph** [comment]
The problem is no matter how I configure the IFC exporter ( extra low) options, it will stuck somewhere "Processing Non spatial elements" at everytime a different element. So I am trying to export IFC from the central project model while someone else from the other end are still editing their design.

**Comment by AngelVelezSosa** [comment]
Right, it sounds like there is some element(s) that Revit is having a hard time (performance-wise) processing.  You can either send us a link to your file and we will try to help or isolate an element that by itself creates a large IFC file.  Likely culprits are long thin objects like rails, walls with many sweeps, reveals and openings, or very complex geometry.

**Comment by AngelVelezSosa** [comment]
There are a number of low-tech and high-tech techniques.
Low-tech: turn off all categories except for one in a 3D view and export only elements in that view.  Does it work?  How big is the IFC file?
High-tech: you can create a script that would export only one element at a time, and look at the size of the IFC files, ranked.

**Comment by jlaiph** [comment]
Thanks. I will run a script then. That is a good idea.
